packages/sources/wbtc-address-set/test/integration/fixtures.ts

```diff
@@ -0,0 +1,90 @@
+import nock from 'nock'
+
+export const mockMembersResponseSuccess = (): nock =>
+  nock('http://localhost:8081', {
+    encodedQueryParams: true,
+  })
+    .get('/')
+    .reply(
+      200,
+      (_, request) => ({
+        result: [
+          {
+            id: '5e5ec5d8a221fb000946fd4ab2d71522',
+            token: 'wbtc',
+            tags: ['dao', 'merchant', 'exchange'],
+            name: 'AirSwap',
+            addresses: [
+              {
+                id: '601323767069d60008cb538a682a0a35',
+                address: '0xfaf0708d1aed2566205d61f471d7e4aeb10910ea',
+                type: 'merchant',
+                balance: '0',
+                verified: true,
+              },
```

Comment:
@boxhock Does this look ok?

Reply:
Didn't expect to see Ethereum addresses here, but the real endpoint does seem to return this as well. It should be fine, because we mainly use the other endpoint anyways.